src/components/Request/FormFields/LinkedinPositionSummary.tsx

```diff
@@ -0,0 +1,47 @@
+import { InfoLabel, Text, Textarea } from "@fluentui/react-components";
+import { Controller } from "react-hook-form";
+import { FormField } from "components/Request/NewRequestForm";
+import "components/Request/Request.css";
+import { TextFieldIcon } from "@fluentui/react-icons-mdl2";
+
+const LinkedinPositionSummary = ({ name, form }: FormField) => {
+  return (
+    <div className="requestFieldContainer">
+      <InfoLabel
+        htmlFor={name + "Id"}
+        weight="semibold"
+        className="requestFieldLabel"
+        required
+        info="A brief summary of the job description (3-4 sentences)"
```

Comment:
AC5 is a little more verbose.

```suggestion
        info="Please provide a brief summary of the job description (3-4 sentences)."
```

Reply:
Sounds good -- was just going based off the AC, but agree with how it was modified and is consistent with other descriptions -- and follows best practices